Templates/SendGrid-Timer-CSharp/run.csx

```diff
@@ -0,0 +1,36 @@
+#r "SendGrid"
+
+using System;
+using SendGrid.Helpers.Mail;
+using Microsoft.Azure.WebJobs.Host;
+
+// The 'From' and 'To' fields are automatically populated with the values specified by the binding settings.
+//
+// You can also optionally configure the default From/To addresses globally via host.config, e.g.:
+//
+// {
+//   "sendGrid": {
+//      "to": "user@host.com",
+//      "from": "Azure Functions <samples@functions.com>"
+//   }
+// }
+public static Mail Run(TimerInfo myTimer, TraceWriter log)
+{
+    log.Info($"C# Timer trigger function executed at: {DateTime.Now}");
+    
+    Mail message = new Mail()
+    {
+        Subject = $"Daily Report"
+    };
+
+	var orderCount = 100;
```

Comment:
Fix formatting. I'd also put a small TODO comment in here that explains that this is where the user should put their business logic. E.g. "TODO: Customize this code to generate your specific mail message"